docs/concepts/storage/persistent-volumes.md

```diff
@@ -695,21 +695,43 @@ parameters:
 
 #### Azure Disk
 
+##### Azure Unmanaged Disk Storage Class (Default)
+
 ```yaml
 kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
-  name: slow
+  name: accounthdd
 provisioner: kubernetes.io/azure-disk
 parameters:
   skuName: Standard_LRS
   location: eastus
   storageAccount: azure_storage_account_name
 ```
 
-* `skuName`: Azure storage account Sku tier. Default is empty.
+* `skuName`: Azure storage account Sku tier. Default is Standard_LRS, allowed values are Standard_LRS and Premium_LRS
 * `location`: Azure storage account location. Default is empty.
-* `storageAccount`: Azure storage account name. If storage account is not provided, all storage accounts associated with the resource group are searched to find one that matches `skuName` and `location`. If storage account is provided, it must reside in the same resource group as the cluster, and `skuName` and `location` are ignored.
+* `storageAccount`: Azure storage account name. If storage account is provided, it must reside in the same resource group as the cluster, and `skuName` and `location` are ignored. If storage account is not provided, a new storage account will be created.
```

Comment:
`If storage account is not provided, a new storage account will be created.` not accurate. 

Should add '.. in the cluster's resource group if no proper storage matches found.